shared/templates/selinux_booleans.csv

```diff
@@ -24,32 +24,32 @@ condor_tcp_network_connect,
 conman_can_network,
 cron_can_relabel,
 cron_system_cronjob_use_shares,
-cron_userdomain_transition,
+cron_userdomain_transition,enable
 cups_execmem,
 cvs_read_shadow,
 daemons_dump_core,
 daemons_enable_cluster_mode,
 daemons_use_tcp_wrapper,
 daemons_use_tty,
-dbadm_exec_content,
+dbadm_exec_content,enable
 dbadm_manage_user_files,
 dbadm_read_user_files,
 deny_execmem,
 deny_ptrace,
 dhcpc_exec_iptables,
 dhcpd_use_ldap,
 docker_connect_any,
-docker_transition_unconfined,
-domain_fd_use,
+docker_transition_unconfined,enable
+domain_fd_use,enable
 domain_kernel_load_modules,
-entropyd_use_audio,
+entropyd_use_audio,disable
```

Comment:
Just OOC why to disable this? Though not much likely there will be audio source on the server, according to e.g.:
https://blog.cloudflare.com/ensuring-randomness-with-linuxs-random-number-generator/ more diversity => stronger entropy (IOW it's enough to XOR entropy source with even weak entropy source to get stronger entropy).

=> I would suggest to enable this (IMHO).